build/scripts/index.sh

```diff
@@ -20,5 +20,11 @@ for meta in "${metas[@]}"; do
     # Ignore double quotes warning for yq expression
     # shellcheck disable=SC2016,SC2094
     cat <<< "$(yq --arg metadir "${META_DIR}" '.links |= . + {self: "/\($metadir)/devfile.yaml" }' "${meta}")"  > "${meta}"
+    if [ "$(yq '.links.v2' "${meta}")" != "null" ]; then
+      # shellcheck disable=SC2016,SC2094
```

Comment:
I think it would be better to fix the issue rather than disabling the check

Reply:
The problem is that `yq` works with expressions in single quotes. We have the same approach above, added description.